### PR TITLE
[view message] Added get messages method #91

### LINKED ISSUE #91: [view messages] Add get_messages method
It will get the messages data from DB and display it in view-messages page in chronological order.
Use Messages table.

### PR DESCRIPTION
**Pull request by morbitton**
fix #91 
I work with @Yarinh55 , she will open PR with the link to html after u approve the HTML page

### CODE REVIEW

**Comment by lmilbaum** [comment]
Should I add Yarin to the list of assignees? If yes, than should add a comment here.

**Comment by lmilbaum** [comment]
@morbitton I am confused. Is this PR WIP or is it ready for review?

**Comment by Yarinh55** [comment]
Hi @lioramilbaum 
we worked together on this PR
It's ready to review and we will add another PR to finish issue #91 after you will approve the messages_view.html #119

**Review by lmilbaum — CHANGES_REQUESTED** [review]
1. This PR has conflicts. Please resolve them.
2. If this PR is a partial fix of  #91, than break it to smaller issues.

**Review by lmilbaum — CHANGES_REQUESTED** [review]
log.log should not be tracked by git. Add it to .gitignore and remove it from this PR.

**Comment by morbitton** [comment]
@lioramilbaum @Yarinh55 
We insert log.log file to .gitignore but we still see the file.
Did we understand u well?

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Adding the file to gitignore does not change after the fact. If you already tracked by git, than gitignore does NOT influence. You should remove the file from the PR. Google it.

**Comment by morbitton** [comment]
@lioramilbaum 
i think its ok now

**Review by lmilbaum — CHANGES_REQUESTED** [review]
what those swo and swp files for?

**Comment by morbitton** [comment]
when we change at the VI file git open those. Nikita said to change file in another way. and we remove those files @lioramilbaum

File: app.py [hunk 1]
```diff
@@ -1,9 +1,42 @@
 #!/usr/bin/env python3
+
 from flask import Flask, render_template, request
+import mysql
+import logging
+# import app_functions
```
Comment (lmilbaum) [inline app.py]: remove comment line

File: app.py [hunk 2]
```diff
@@ -1,9 +1,42 @@
...
+    return connection
+
+
+def view_messages():
+    connection = connect_db()
+    message = connection.cursor()
+    message.execute("SELECT * FROM tiger.messages")
+    view = message.fetchall()
+    # for row in view will change after implement of the view_massages.html
+    # print(row)
```
Comment (lmilbaum) [inline app.py]: remove comment

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (morbitton) [reply app.py]: delete for now because the file delete

File: app.py (hunk 2 above)
Comment (morbitton) [reply app.py]: done

**Review by lmilbaum — APPROVED** [review]
LGTM

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Have you pulled the latest changes and rebased?

**Comment by morbitton** [comment]
yes , everytime Im get the git i run the comment "git pull upstream"

**Comment by lmilbaum** [comment]
You need to pull from upstream master and then rebase your branch

**Comment by morbitton** [comment]
now i see there is conflict, 
work on it and let u know as we finish , sorry

**Review by lmilbaum — APPROVED** [review]
LGTM

**Review by lmilbaum — CHANGES_REQUESTED** [review]
still conflicts

**Comment by morbitton** [comment]
image

@lioramilbaum run all the conflict. git said to me im Successfully rebased and updated refs/heads/test.

**Comment by lmilbaum** [comment]
Look at the instructions I have posted in Saar's PR.
https://github.com/Code-Collaborate-Community/tiger/pull/120#issuecomment-572005291

**Pull request closed without merging**